src/frontend/context/BrevModulContext.ts

```diff
@@ -19,56 +19,71 @@ import {
     ISelectOptionMedBrevtekst,
 } from '../komponenter/Felleskomponenter/Hendelsesoversikt/BrevModul/typer';
 import { AxiosError } from 'axios';
-import { feil, IFelt, nyttFelt, ok, Valideringsmetadata } from '../typer/felt';
-import { useSkjema } from '../typer/skjema';
+import { useSkjema } from '../familie-skjema/skjema';
 import { fjernWhitespace } from '../utils/commons';
 import { IGrunnlagPerson } from '../typer/person';
 import { Målform } from '../typer/søknad';
+import { useFelt } from '../familie-skjema/felt';
+import { FeltState, FeltContext, Valideringsstatus } from '../familie-skjema/typer';
+import { feil, ok } from '../familie-skjema/validators';
 
 const [BrevModulProvider, useBrevModul] = createUseContext(() => {
     const { axiosRequest } = useApp();
     const { åpenBehandling } = useBehandling();
-    const {
-        hentFeltProps,
-        kanSendeSkjema,
-        nullstillFelt,
-        onSubmit,
-        oppdaterFeltISkjema,
-        skjema,
-    } = useSkjema<IFagsak>({
+
+    const mottakerIdent = useFelt({
+        verdi: '',
+        valideringsfunksjon: (felt: FeltState<string>) =>
+            felt.verdi.length >= 1 ? ok(felt) : feil(felt, 'Du må velge en mottaker'),
+    });
+    const brevmal = useFelt({
+        verdi: '',
+        valideringsfunksjon: (felt: FeltState<Brevmal | ''>) =>
+            felt.verdi ? ok(felt) : feil(felt, 'Du må velge en brevmal'),
+    });
+    const multiselect = useFelt({
+        verdi: [],
+        valideringsfunksjon: (
+            felt: FeltState<ISelectOptionMedBrevtekst[]>,
+            avhengigheter?: FeltContext
+        ) => {
+            const brevmal: Brevmal | '' = avhengigheter?.brevmal.value;
```

Comment:
For å unngå misforståelser er det kanskje lurt å prefikse ` ... = useFelt ` med `Felt` => `brevmalFelt`

Reply:
Men her henter jeg jo ut den faktiske verdien? Isåfall må jeg endre i avhengighetene og definere de slik:
```
{
    brevmalFelt: brevmal
}
```